Playwright.Axe/AxeFrameContext.cs

```diff
@@ -0,0 +1,31 @@
+﻿#nullable enable
+
+namespace Playwright.Axe
+{
+    /// <summary>
+    /// Run Context for a Frame
+    /// </summary>
+    public sealed class AxeFrameContext
+    {
+        /// <summary>
+        /// Selector for the Frame in question.
+        /// </summary>
+        public AxeCrossTreeSelector FrameSelector { get; }
```

Comment:
`AxeCrossTreeSelector` feels like an unnecessarily long name - I think just `AxeSelector` would be sufficient here, this uses the same `string | string[]` representation that all axe selector properties use. There are actually a few other points in this library that probably ought to be based on an `AxeSelector` type like this - for example, `AxeResultNode`'s `Target` property. It might make more sense to hold off on this PR until after we merge into the monorepo so we can just reuse the similar typings that `Selenium.Axe` already has for this